CONTRIBUTING.md

```diff
@@ -393,10 +393,14 @@ You can find documentation style guidelines in [RFC 1574][rfc1574].
 
 [rfc1574]: https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text
 
-In many cases, you don't need a full `./x.py doc`. You can use `rustdoc` directly
-to check small fixes. For example, `rustdoc src/doc/reference.md` will render
-reference to `doc/reference.html`. The CSS might be messed up, but you can
-verify that the HTML is right.
+In many cases, you don't need a full `./x.py doc`, which will build the entire
+stage 2 compiler. When updating documentation for the standard library, first
+try `./x.py doc --stage 0 src/libstd`. Results will should appear in
+`build/$TARGET/crate-docs`.
```

Comment:
We should definitely recommend stage 1 (which should always work) if this fails, which it could, as rustdoc doesn't run on stage 0 in CI, so some annotations may be missing or otherwise incompatible with stage 0. Lack of effective cfg(bootstrap) for docs makes that a particularly thorny problem.